openlibrary/core/bookshelves.py

```diff
@@ -255,3 +255,14 @@ def user_with_most_books(cls):
                  "ORDER BY counted DESC, username LIMIT 100")
         result = oldb.query(query)
         return list(result)
+
+    @classmethod
+    def migrate_bookshelves(cls, from_work_id: int, to_work_id: int):
+        oldb = db.get_db()
+        query = "UPDATE bookshelves_books SET work_id = $to_work_id " \
+                "WHERE work_id = $from_work_id RETURNING *;"
```

Comment:
PEP8 encourages us to avoid backslash line continuation because whitespace to the right of the \ will break the script on a change that is invisible to the reader.
```suggestion
        query = ("UPDATE bookshelves_books SET work_id = $to_work_id "
                 "WHERE work_id = $from_work_id RETURNING *;")
```